docs/ota-client-guide/modules/ROOT/pages/setup-boot-image-for-ostree.adoc

```diff
@@ -0,0 +1,44 @@
+= Set up boot image layout for OSTree compatibility
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+include::_partials/aktualizr-version.adoc[]
+
+This is the second step in getting a new board running after xref:add-board-class.adoc[adding a new board class for the new target board in meta-updater].
+
+This step involves getting Yocto to build a physical image with the partitions in the correct place, with the necessary modifications for OSTree. The standard approach here is to use https://www.yoctoproject.org/docs/{yocto-version}/dev-manual/dev-manual.html#creating-partitioned-images-using-wic[Wic, the OpenEmbedded Image Creator].
+
+You will need to create a Wic Kickstart file that places the necessary firmware in memory where the board needs it to be. Typically, board vendors will provide a table with the memory layout of the board’s firmware.
+
+Once you complete this step, you can proceed to the final step of xref:add-meta-updater-to-vendors-sdk.adoc[adding meta-updater features to the vendor's SDK].
+
+
+For example, using our NXP board example, the layout was provided in the https://www.nxp.com/docs/en/user-guide/LSDKUG_Rev19.06.pdf[Landscape SDK User Guide], Chapter 4.2, tables 14 and 16.
+
+.Table of NXP memory layout
+image::img::screenshot_lsdk_memory_layout.png[width=100%]
+
+.Table of NXP default layout
+image::img::screenshot_lsdk_default_layout.png[width=100%]

```

Comment:
Ah, I see what you mean now in https://github.com/advancedtelematic/ota-connect-docs-images/pull/2#issuecomment-586282298. Yeah, makes sense to have both these images here.